src/mixins/auth-api-mixin.ts

```diff
@@ -0,0 +1,63 @@
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { axios } from '@/utils'
+
+// Interfaces
+import { BusinessContactIF } from '@/interfaces'
+/**
+ * Mixin that provides integration with the Legal API.
+ */
+@Component({})
+export default class AuthApiMixin extends Vue {
+  @Getter getBusinessId!: string
+
+  /**
+   * Fetches contact information for the specified business.
+   * @returns a promise to return the contact data
+   */
+  async getContactInfo (): Promise<any> {
+    if (!this.getBusinessId) throw new Error('Invalid parameter \'businessIdentifier\'')
+
+    const url = `entities/${this.getBusinessId}`
+    const authUrl = sessionStorage.getItem('AUTH_API_URL')
+    const config = { baseURL: authUrl }
+
+    return axios.get(url, config)
+      .then(response => {
+        if (response && response.data && response.data.contacts) {
+          // Always take the first contact.
+          return {
+            ...response.data.contacts[0],
+            confirmEmail: response.data.contacts[0].email,
+            extension: response.data.contacts[0].phoneExtension
+          } || []
```

Comment:
return the object or the empty array?

Reply:
Good point, i believe we are deconstructing down to `contacts` which itself is an array but since we are always returning the first object in that array it will never return an array under normal circumstances. 
I will look into this!